.evergreen/generate_evergreen_tasks.js

```diff
@@ -537,7 +537,7 @@ const oneOffFuncAsTasks = oneOffFuncs.map(oneOffFunc => ({
 }));
 
 oneOffFuncAsTasks.push({
-  name: 'run-custom-csfle-tests-pinned-commit',
+  name: 'run-custom-csfle-tests-pinned-commit-mongocryptd',
```

Comment:
```suggestion
  name: 'run-custom-csfle-tests-mongocryptd-pinned-commit',
```

non-blocking - this is just for consistency with the shared library tests.